packages/Datadog.Unity/README.md

```diff
@@ -10,16 +10,14 @@ The Datadog Unity SDK supports logging and crash reporting for Android and iOS a
 
 1. Install [External Dependency Manager for Unity (EDM4U)](https://github.com/googlesamples/unity-jar-resolver). This can be done using [Open UPM](https://openupm.com/packages/com.google.external-dependency-manager/).
 
-2. Disable static linking for iOS in EDM4U. This can be found in Assets -> External Dependency Manager -> iOS Resolver -> Settings -> Link frameworks statically. If this option is left enabled, you will see errors that Datadog was not initialized when attempting to create loggers or start views.
-
-3. Add the Datadog SDK Unity package from its Git URL at [https://github.com/DataDog/unity-package](https://github.com/DataDog/unity-package).  The package url is `https://github.com/DataDog/unity-package.git`.
+2. Add the Datadog SDK Unity package from its Git URL at [https://github.com/DataDog/unity-package](https://github.com/DataDog/unity-package).  The package url is `https://github.com/DataDog/unity-package.git`.
 
 > [!NOTE]
-> Datadog plans on adding support for Open UPM after Closed Beta.
+> Datadog plans on adding support for Open UPM after Beta.
```

Comment:
We don't document the future. Please delete this.

Reply:
This only commenting on where / how the plugin will distributed post beta, not really a feature. I would like to keep it as it is a common question we will get.

Reply:
Okay. I still advise against it (it may lead to more questions, etc.), but as it's not in our official docs, you can keep it in if you really want to.